code/datums/mood_events/generic_positive_events.dm

```diff
@@ -272,3 +272,24 @@
 	mood_change = 10
 	timeout = 5 MINUTES
 
+/datum/mood_event/won_52_card_pickup
+	description = "HA! That loser will be picking cards up for a long time!"
+	mood_change = 3
+	timeout = 3 MINUTES
+
+/datum/mood_event/playing_cards
+	description = "I'm enjoying playing cards with other people!"
+	mood_change = 2
+	timeout = 3 MINUTES
+
+/datum/mood_event/playing_cards/add_effects(param)
+	var/card_players = 1
+	for(var/mob/living/carbon/player in viewers(COMBAT_MESSAGE_RANGE, owner))
+		var/player_has_cards = player.is_holding(/obj/item/toy/singlecard) || player.is_holding(/obj/item/toy/cards/deck) || player.is_holding(/obj/item/toy/cards/cardhand)
```

Comment:
you want `is_holding_item_of_type()` for typepaths

Reply:
Applied with https://github.com/tgstation/tgstation/pull/64200/commits/e76ab3117f9e466765724c196f13c8ac3f0001ac